tough/src/editor/signed.rs

```diff
@@ -21,7 +21,12 @@ use serde_plain::forward_from_str_to_serde;
 use snafu::{ensure, OptionExt, ResultExt};
 use std::collections::HashMap;
 use std::fs;
+
+#[cfg(not(target_os = "windows"))]
 use std::os::unix::fs::symlink;
+#[cfg(target_os = "windows")]
+use std::os::windows::fs::symlink_file;
```

Comment:
```suggestion
use std::os::windows::fs::symlink_file as symlink;
```



Since these two versions of the functions have the same signature, we can rename the `symlink_file` to `symlink` which I believe will eliminate the need to have `cfg` statements elsewhere in this file.

Could you try making this change and undoing the other changes in this file to see if it works?

Reply:
Applied the suggestion in the #174d5cf commit